internal/commands/vorpal-engine.go

```diff
@@ -0,0 +1,125 @@
+package commands
+
+import (
+	"path/filepath"
+
+	"github.com/checkmarx/ast-cli/internal/commands/util/printer"
+	errorConstants "github.com/checkmarx/ast-cli/internal/constants/errors"
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+func runScanVorpalCommand() func(cmd *cobra.Command, args []string) error {
+	return func(cmd *cobra.Command, args []string) error {
+		vorpalLatestVersion, _ := cmd.Flags().GetBool(commonParams.VorpalLatestVersion)
+		sourceFlag, _ := cmd.Flags().GetString(commonParams.SourcesFlag)
+
+		scanResult, err := ExecuteVorpalScan(sourceFlag, vorpalLatestVersion)
+		if err != nil {
+			return err
+		}
+
+		err = printer.Print(cmd.OutOrStdout(), scanResult, printer.FormatJSON)
```

Comment:
is print command is instead of return?

Reply:
yes, 
the command returns only an error
the return value is printed to the cmd output stream

Reply:
In general when an error is returned from to the command function it's printed to the console.